docs/user-guide/integrations.md

```diff
@@ -13,7 +13,9 @@ redirect_from: "/docs/integrations/"
     * [Build Next](https://github.com/albertosantini/sublimetext-buildnext)
 * [Vim](https://github.com/scrooloose/syntastic/tree/master/syntax_checkers/javascript)
 * Emacs: [Flycheck](http://flycheck.readthedocs.org/en/latest/) supports ESLint in recent versions.
-* Eclipse Orion: ESLint is the [default linter](http://dev.eclipse.org/mhonarc/lists/orion-dev/msg02718.html)
+* Eclipse 
+    * Eclipse Orion: ESLint is the [default linter](http://dev.eclipse.org/mhonarc/lists/orion-dev/msg02718.html)
```

Comment:
I don't think there's a need for a sublist here, as these two editors have nothing in common other than a piece of their name. Can you just add one new bullet item to the list?